### ISSUE TITLE
Could not load file or assembly LumenWorks.Framework.IO

### ISSUE THREAD

**Issue by ferenczy** [zarunbal/LogExpert#49]
When I try to select the columnizer _CSV columnizer_, I get the following exception:

```
Could not load file or assembly 'LumenWorks.Framework.IO, Version=3.0.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. A strongly-named assembly is required. (Exception from HRESULT: 0x80131044)

FileLoadException
   at CsvColumnizer.CsvColumnizer.Selected(ILogLineColumnizerCallback callback)
   at CsvColumnizer.CsvColumnizer.Configure(ILogLineColumnizerCallback callback, String configDir)
   at LogExpert.FilterSelectorForm.configButton_Click(Object sender, EventArgs e)
   at System.Windows.Forms.Control.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ButtonBase.WndProc(Message& m)
   at System.Windows.Forms.Button.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

The DLL `LumenWorks.Framework.IO.dll` is present in the `plugins` directory and has the version 3.0.0.0.

The LogExpert version is 1.6.7 (portable from an archive, i.e. _not installed_) running on Windows 7 Ultimate 64-bit. The file is from a local drive.

The log file doesn't contain more information than you may find in the exception I posted.

**Comment by zarunbal** [comment]
Thanks for your issue. The important part is at the end of the exception text, `A strongly-named assembly is required.` I started to sign the LogExpert assemblies but missed the csv columnizers which references a unsigned dll. I have to check the available version of Lumenworks and maybe update the references.

**Comment by zarunbal** [comment]
The fix is in commit ad553d3c1642be264b9df1afe9f7c70c283feb47 the CI build is available under [appveyor](https://ci.appveyor.com/project/Zarunbal/logexpert/build/0.0.110) (This build should work but I can't give you a warranty ;) )

**Comment by zarunbal** [comment]
Release 1.6.13 is out there with the fix. I close this issue. If there is still a problem feel free to reopen this issue.
